docker-compose.uffizzi.yml

```
@@ -0,0 +1,15 @@
+version: "3"
+x-uffizzi:
+  ingress:
+    service: livebook
+    port: 8080
+services:
+  livebook:
+    image: "${APP_IMAGE}"
+    environment:
+      - LIVEBOOK_PORT=8080
+      - LIVEBOOK_BASE_URL=/livebook
+      - LIVEBOOK_PASSWORD=livebookisawesome
```

@jonatanklosko with or without password?

```suggestion
      - LIVEBOOK_TOKEN_ENABLED=false
```

---

without password is good